javascript/javascript1/week4/formatternum.js

```diff
@@ -0,0 +1,19 @@
+function formatCreditCardNumber(number) {
+   // Check if input is a valid number
+   if (typeof number !== 'number' || isNaN(number)) {
+       return "This is not a number";
+   }
+
+   const charsNumber = number.toString().split('');
+   const len = charsNumber.length;
```

Comment:
Avoid variable names that are abbreviations. They might make sense when you write the code, but if someone else reads it, they might not make as much sense, as well if you come back to the project/file in a few months.
`cardNumberLength` could be better